contracts/nns/contract.go

```diff
@@ -508,7 +508,7 @@ func SetRecord(name string, typ recordtype.Type, id byte, data string) {
 	if recBytes == nil {
 		panic("invalid record id")
 	}
-	storeRecord(ctx, recordKey, name, typ, id, data)
+	storeRecord(ctx, tokenID, name, typ, id, data)
```

Comment:
`getIdRecordKey` redundantly done twice now, not an improvement for me

Reply:
This opens up path for 78862e4322fb2b2235a51c4666af94f1fb113a23.